chromadb/test/ef/test_multimodal_ef.py

```diff
@@ -1,33 +1,17 @@
-from typing import Generator, cast
+from typing import Generator
 import numpy as np
 import pytest
 import chromadb
 from chromadb.api.types import (
     Embeddable,
     EmbeddingFunction,
-    Embeddings,
     Image,
     Document,
 )
-from chromadb.test.property.strategies import hashing_embedding_function
+from chromadb.test.property.strategies import hashing_multimodal_ef
 from chromadb.test.property.invariants import _exact_distances
 
 
-# A 'standard' multimodal embedding function, which converts inputs to strings
-# then hashes them to a fixed dimension.
-class hashing_multimodal_ef(EmbeddingFunction[Embeddable]):
```

Comment:
Because of the way tests load modules, this would previously have caused the registry to emit an error that it was being imported twice. Moving it to the strategies / fixtures fixes this.